appveyor.yml

```diff
@@ -30,10 +33,10 @@ install:
 
 test_script:
   # we don't run the "test phase" when doing deploys
-  - if [%APPVEYOR_REPO_TAG%]==[false] ( cargo +%CHANNEL%-%TARGET% build )
-  - if [%APPVEYOR_REPO_TAG%]==[false] ( cargo +%CHANNEL%-%TARGET% test )
+  - if [%APPVEYOR_REPO_TAG%]==[false] ( cargo +%CHANNEL%-%TARGET% build --all )
```

Comment:
So there were a lot of commits with a lot of failures.  Do you happen to remember what failure you were resolving by adding this?

Reply:
I'll track back the errors, I forgot to keep a link -_-